### PR TITLE
Fix README links

### PR DESCRIPTION
**Pull request by loichuder**
Before merging, I would like to discuss a few issues in the README. See the comments below.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![Github Actions Status](https://github.com/jupyterlab/mimerenderer-cookiecutter-ts/workflows/CI/badge.svg)
 
-A [cookiecutter](https://github.com/audreyr/cookiecutter) template for creating
+A [cookiecutter](https://github.com/cookiecutter/cookiecutter) template for creating
 a JupyterLab MIME renderer extension in TypeScript. (See also
 [mimerender-cookiecutter](https://github.com/jupyterlab/mimerender-cookiecutter)
 for a MIME renderer extension in CommonJS.)
```
Comment (loichuder) [inline README.md:8]: Should the link to this archived repository be kept ?

File: README.md [hunk 2]
```diff
@@ -24,8 +24,8 @@ cookiecutter https://github.com/jupyterlab/mimerender-cookiecutter-ts
 If you'd like to generate a package for a specific JupyterLab release, use the `--checkout` option and give a tag or commit from this repository.
 
 ```
-cookiecutter https://github.com/jupyterlab/extension-cookiecutter-ts --checkout v1.0
-cookiecutter https://github.com/jupyterlab/extension-cookiecutter-ts --checkout v2.0
+cookiecutter https://github.com/jupyterlab/mimerender-cookiecutter-ts --checkout v1.0
+cookiecutter https://github.com/jupyterlab/mimerender-cookiecutter-ts --checkout v2.0
```
Comment (loichuder) [inline README.md:28]: Actually, there is no `v2.0` tag for this repo yet. As the master branch is now supporting JupyterLab 3, it should be set to a previous commit.

File: README.md (hunk 1 above)
Comment (blink1073) [reply README.md:8]: Thanks for catching that, the new link sounds good.

File: README.md (hunk 2 above)
Comment (blink1073) [reply README.md:28]: I created the v2.0 tag, thanks for the heads up!

**Comment by blink1073** [comment]
Thanks for the fixes!  We picked up a merge conflict from #25, mind rebasing?

**Comment by loichuder** [comment]
Rebased and ready to be merged !

**Review by blink1073 — APPROVED** [review]
Thanks again!

**Pull request merged**